clients/fides-js/src/lib/initialize.ts

```diff
@@ -143,6 +146,35 @@ const automaticallyApplyGPCPreferences = ({
   }
 };
 
+/**
+ * Gets and validates Fides override options provided through URL query params, cookie or window obj.
+ */
+export const getOverrideFidesOptions = (): FidesOptionOverrides => {
+  const overrideOptions: Partial<FidesOptionOverrides> = {};
+  if (typeof window !== "undefined") {
+    const params = new URLSearchParams(document.location.search);
+    FIDES_OVERRIDE_OPTIONS_VALIDATOR_MAP.forEach(
+      ({ fidesOption, fidesOptionType, fidesOverrideKey, validationRegex }) => {
+        // look for override options on URL query params, window obj, and cookie
+        const queryParamOverride: string | null = params.get(fidesOverrideKey);
+        const windowObjOverride: string | boolean | undefined = window.config
+          ?.fides
+          ? window.config?.fides[fidesOverrideKey]
+          : undefined;
+        const cookieOverride: string | undefined =
```

Comment:
I don't think you necessarily need to declare these types—TS should be able to figure this out itself

Reply:
I'd rather be explicit here with types, for future dev work. Easier to parse what kinds of data we're working with